src/components/calcite-date-day/calcite-date-day.scss

```diff
@@ -40,7 +51,14 @@
 }
 
 :host([scale="m"]) {
-  padding: 4px 4px;
+  .day-v-wrapper {
+    padding-top: 4px;
+    padding-bottom: 4px;
+  }
+  .day-wrapper {
+    padding-left: 4px;
```

Comment:
We've just recently added tailwind to this repo, so I think we could get these values from the spacing scale with padding modifiers and and the `@apply` directive like:

```

:host([scale="s"]) {
  .day-v-wrapper {
    padding-top: 2px;
    padding-bottom: 2px;
  }
  .day-wrapper {
    @apply px-0;
  }
  .day {
    height: 27px;
    width: 27px;
    @apply text--2;
  }
}

:host([scale="m"]) {
  .day-v-wrapper {
    @apply py-1;
  }
  .day-wrapper {
    @apply px-1;
  }
  .day {
    height: 33px;
    width: 33px;
    @apply text--1;
  }
}

:host([scale="l"]) {
  .day-v-wrapper {
    @apply py-1;
  }
  .day-wrapper {
    @apply px-1;
  }
  .day {
    height: 43px;
    width: 43px;
    @apply text-0;
  }
}
```

This hasn't made it to our conventions doc, but if you want to be a trail blazer you could play around with it.

I'm also unsure as to how you would express `2px` or the other units in tailwind-speak. It's pretty new to me...

Reply:
Awesome, let me take a look at this.

Reply:
@paulcpederson - a general question on this.  For things like 
```css
:host {
  display: inline-block;
  vertical-align: top;
  width: 100%;
  position: relative;
  overflow: visible;
}
```
Do we want to replace it completely with tailwind like below, or are we ok with what we had?

```
:host {
  @apply inline-block align-top w-full relative overflow-visible;
}
```